### PR TITLE
Vehicles interface design

### PR DESCRIPTION
**Pull request by jlyon1**

### CODE REVIEW

File: api/vehicles.go [hunk 1]
```diff
@@ -62,14 +62,40 @@ func (api *API) VehiclesCreateHandler(w http.ResponseWriter, r *http.Request) {
...
+
+	if err != nil {
+		http.Error(w, err.Error(), http.StatusInternalServerError)
+		return;
+	}
+	vehicle.Updated = time.Now()
+	err = api.db.Vehicles.Update(bson.M{"vehicleID": vehicle.VehicleID}, vehicle)
+	if err != nil {
+		http.Error(w, err.Error(), http.StatusInternalServerError)
+		return;
```
Comment (kochman) [inline api/vehicles.go]: No semicolon

File: api/vehicles.go [hunk 2]
```diff
@@ -62,14 +62,40 @@ func (api *API) VehiclesCreateHandler(w http.ResponseWriter, r *http.Request) {
 }
 
 func (api *API) VehiclesEditHandler(w http.ResponseWriter, r *http.Request) {
+	if api.cfg.Authenticate && !cas.IsAuthenticated(r) {
+		return
+	}
+	vehicle := model.Vehicle{}
+	// vars := mux.Vars(r)
```
Comment (kochman) [inline api/vehicles.go]: Necessary?

File: admin.html [hunk 3]
```diff
@@ -19,9 +20,13 @@
     <div class ="navbar">
       <p class="title">RPI Shuttle Tracker</p>
       <a class ="logout button" href="/admin/logout/">Logout</a>
-      <div class ="button logout" onclick="Admin.showMapPanel();">Create Route</div>
+      <div class ="button logout" onclick="Admin.toggleDisplay();">Toggle Route Display</div>
+      <div class ="button logout rtb" onclick="Admin.showMapPanel();">Create Route</div>
```
Comment (kochman) [inline admin.html]: All of the element attributes should have no spacing around the `=`, i.e. `<div class="button...">`

File: static/js/vehicles.js [hunk 4]
```diff
@@ -0,0 +1,127 @@
+$(document).ready(function(){
+  console.log("asdf");
+});
+
+refresh = true;
+
+Vue.component('vehicle-create',{
+  template:`<div class="vehicle-card route-description-box">
+    <b>id</b>: <input type="textbox" v-model="id" placeholder="1123454125"></input> (must be same as itrak vehicle id) <br>
+    <b>name</b>:<input type="textbox" v-model="name" placeholder="Joey's Really cool vehicle"></input> <br>
```
Comment (kochman) [inline static/js/vehicles.js]: Let's make this "Vehicle name" or something

File: static/js/vehicles.js [hunk 5]
```diff
@@ -0,0 +1,127 @@
+$(document).ready(function(){
+  console.log("asdf");
+});
+
+refresh = true;
+
+Vue.component('vehicle-create',{
+  template:`<div class="vehicle-card route-description-box">
+    <b>id</b>: <input type="textbox" v-model="id" placeholder="1123454125"></input> (must be same as itrak vehicle id) <br>
```
Comment (kochman) [inline static/js/vehicles.js]: maybe "ID" and "(must be the same as iTrak's vehicle ID)"

**Comment by jlyon1** [comment]
not ready

**Comment by jlyon1** [comment]
ready

**Pull request merged**